mindsdb/integrations/handlers/langchain_handler/langchain_handler.py

```diff
@@ -59,7 +60,9 @@ def __init__(self, *args, **kwargs):
         self.default_mode = 'default'  # can also be 'conversational' or 'conversational-full'
         self.engine_to_supported_modes = {
             'openai': ['default', 'conversational', 'conversational-full', 'image'],
-            'anthropic': ['default', 'conversational', 'conversational-full']
+            'anthropic': ['default', 'conversational', 'conversational-full'],
```

Comment:
Out of curiosity, why do `anyscale` and `litellm` not support `conversational-full`? AFAIK if a provider supports one, it should be able to support the other.

Reply:
I think it should support the 'default' mode also. But I can't find where 'conversational-full' is used?

Reply:
Ah, you may be right. I won't block the PR on this, feel free to keep it as-is.